include/z64elf_message.h

```diff
@@ -0,0 +1,129 @@
+#ifndef Z64ELF_MESSAGE_H
+#define Z64ELF_MESSAGE_H
+
+#include "ultra64.h"
+
+// Checks the condition and exits the script if the check passes
+#define ELF_MESSAGE_TYPE_CHECK      0
+// ? (unused)
+#define ELF_MESSAGE_TYPE_UNK_1      1
+// ? (unused)
+#define ELF_MESSAGE_TYPE_UNK_2      2
+// Checks the condition and branches to another command if the check passes
+#define ELF_MESSAGE_TYPE_BRANCHING  3
```

Comment:
Also this is more of a general point, but I don't know if "branching" is the right term since afaict it only skips a certain number of commands as opposed to going to a specific one (see the relevant case in `ElfMessage_GetTextFromMsgs`)